deployments/examples/ocis_individual_services/docker-compose.yml

```diff
@@ -42,6 +53,154 @@ services:
+      APPPROVIDER_LOG_PRETTY: "${OCIS_LOG_PRETTY:-false}"
+
+      APP_PROVIDER_GRPC_ADDR: 0.0.0.0:9164
+      APP_PROVIDER_EXTERNAL_ADDR: app-provider:9164
+
+      REVA_GATEWAY: gateway:9142
+      APP_PROVIDER_JWT_SECRET: ${OCIS_JWT_SECRET:-Pive-Fumkiu4}
+
+  notifications:
+    image: owncloud/ocis:${OCIS_DOCKER_TAG:-latest}
+    deploy:
+      replicas: ${OCIS_SCALE:-1}
+    networks:
+      ocis-net:
+    entrypoint:
+      - ocis
+      - notifications
+      - server
+    environment:
+      NOTIFICATIONS_LOG_LEVEL: "${OCIS_LOG_LEVEL:-error}"
+      NOTIFICATIONS_LOG_COLOR: "${OCIS_LOG_COLOR:-false}"
+      NOTIFICATIONS_LOG_PRETTY: "${OCIS_LOG_PRETTY:-false}"
+
+      REVA_GATEWAY: gateway:9142
+      NOTIFICATIONS_EVENTS_ENDPOINT: nats:9233
+      NOTIFICATIONS_MACHINE_AUTH_API_KEY: ${OCIS_MACHINE_AUTH_API_KEY:-changeme}
+
+  idm:
+    image: owncloud/ocis:${OCIS_DOCKER_TAG:-latest}
+    deploy:
+      replicas: ${OCIS_SCALE:-1}
+    networks:
+      ocis-net:
+    entrypoint:
+      - ocis
+      - idm
+      - server
+    environment:
+      IDM_LOG_LEVEL: "${OCIS_LOG_LEVEL:-error}"
+      IDM_LOG_COLOR: "${OCIS_LOG_COLOR:-false}"
+      IDM_LOG_PRETTY: "${OCIS_LOG_PRETTY:-false}"
+
+      IDM_LDAPS_ADDR: 0.0.0.0:9235
+      IDM_ADMIN_USER_ID: ${ADMIN_USER_ID:-09246a85-682a-4cd5-996d-8e8d2aca50af}
+      IDM_ADMIN_PASSWORD: "admin" #TODO: change me
+      IDM_SVC_PASSWORD: ${IDM_SVC_PASSWORD:-changeme}
+      IDM_REVASVC_PASSWORD:  ${IDM_REVASVC_PASSWORD:-changeme}
+      IDM_IDPSVC_PASSWORD:  ${IDM_IDPSVC_PASSWORD:-changeme}
+
+
+      IDM_CREATE_DEMO_USERS: ${DEM_USERS:-true}
+
+      IDM_JWT_SECRET: ${OCIS_JWT_SECRET:-Pive-Fumkiu4}
+
+  ocdav:
+    image: owncloud/ocis:${OCIS_DOCKER_TAG:-latest}
+    deploy:
+      replicas: ${OCIS_SCALE:-1}
+    networks:
+      ocis-net:
+    entrypoint:
+      - ocis
+      - ocdav
+      - server
+    environment:
+      OCDAV_LOG_LEVEL: "${OCIS_LOG_LEVEL:-error}"
+      OCDAV_LOG_COLOR: "${OCIS_LOG_COLOR:-false}"
+      OCDAV_LOG_PRETTY: "${OCIS_LOG_PRETTY:-false}"
+
+      OCDAV_HTTP_ADDR: 0.0.0.0:8080
+
+      OCDAV_PUBLIC_URL: ${OCIS_DOMAIN:-ocis.owncloud.test}
+
+      REVA_GATEWAY: gateway:9142
+      OCDAV_JWT_SECRET: ${OCIS_JWT_SECRET:-Pive-Fumkiu4}
+  graph-explorer:
+    image: owncloud/ocis:${OCIS_DOCKER_TAG:-latest}
+    deploy:
+      replicas: ${OCIS_SCALE:-1}
+    networks:
+      ocis-net:
+    entrypoint:
+      - ocis
+      - graph-explorer
+      - server
+    environment:
+      GRAPH_EXPLOER_LOG_LEVEL: "${OCIS_LOG_LEVEL:-error}"
+      GRAPH_EXPLOER_LOG_COLOR: "${OCIS_LOG_COLOR:-false}"
+      GRAPH_EXPLOER_LOG_PRETTY: "${OCIS_LOG_PRETTY:-false}"
+
+      GRAPH_EXPLORER_HTTP_ADDR: 0.0.0.0:9135
+      GRAPH_EXPLORER_ISSUER: https://${OCIS_DOMAIN:-ocis.owncloud.test}
+      GRAPH_EXPLORER_GRAPH_URL_BASE: https://${OCIS_DOMAIN:-ocis.owncloud.test}
+
+  audit:
+    image: owncloud/ocis:${OCIS_DOCKER_TAG:-latest}
+    deploy:
+      replicas: ${OCIS_SCALE:-1}
```

Comment:
should we really scale the audit service?

Reply:
probably not